requirements.txt

```diff
@@ -1,4 +1,5 @@
 # note there are also rpm packaged deps that aren't
 # setup in pypi or with setup.py
 ../python-rhsm
+decorator
```

Comment:
Since we're pulling this in with `Requires` in the spec, don't see much point in keeping it here.